remo/settings/base.py

```diff
@@ -300,10 +298,6 @@ def COMPRESS_JINJA2_GET_ENVIRONMENT():
 # Paginator settings
 ITEMS_PER_PAGE = 20
 
-# django-celery setup
-import djcelery  # noqa
-djcelery.setup_loader()
```

Comment:
Yes, these will be removed from the commit once it's rebased (see initial comment). If I leave the djcelery parts, celery4 won't work.